### PR TITLE
fix(storage): reduce tsm level metrics cardinality

### PR DESCRIPTION
**Pull request by tmgordeeva**
This should have cut off TSM file levels at 4+.

### CODE REVIEW

File: tsdb/tsm1/file_store.go [hunk 1]
```diff
@@ -339,22 +343,29 @@ func (t *fileTracker) AddBytes(bytes uint64, level int) {
...
+			level = 5
+		}
+		labels["level"] = fmt.Sprintf("%d", level)
+		if level > maxLevel {
+			maxLevel = level
 		}
 		t.metrics.Files.With(labels).Set(float64(v))
 	}
-	atomic.StoreUint64(&t.levels, level)
+	atomic.StoreUint64(&t.levels, maxLevel)
```
Comment (e-dard) [inline tsdb/tsm1/file_store.go]: What is `t.levels` used for? It's never returned from anywhere? Is this a regression that's been introduced?

File: tsdb/tsm1/file_store.go [hunk 2]
```diff
@@ -339,22 +343,29 @@ func (t *fileTracker) AddBytes(bytes uint64, level int) {
...
+	maxLevel := uint64(0)
 	for k, v := range files {
-		labels["level"] = fmt.Sprintf("%d", k)
-		if uint64(k) > level {
-			level = uint64(k)
+		level := uint64(k)
+		if level > 5 {
+			level = 5
+		}
+		labels["level"] = fmt.Sprintf("%d", level)
```
Comment (e-dard) [inline tsdb/tsm1/file_store.go]: If the level is 5 or over then the label should be `5+` really.

Comment (dgnorton) [reply tsdb/tsm1/file_store.go]: nit: `6+` since it's `if level > 5`

File: tsdb/tsm1/file_store.go (hunk 1 above)
Comment (tmgordeeva) [reply tsdb/tsm1/file_store.go]: Used to keep track of metrics to clear later, but we could equally just set all the levels we care about to 0.

Comment (e-dard) [reply tsdb/tsm1/file_store.go]: Yeah I don't understand what it's for. We should clear this up.

**Comment by benbjohnson** [comment]
It looks like this does levels `1`, `2`, `3`, `4`, & `5+`. Should it just go up to `4+` instead?

**Comment by tmgordeeva** [comment]
Now goes to 4+ instead of 5+.

**Pull request merged**